cavern/src/main/java/org/opencadc/cavern/files/CavernURLGenerator.java

```diff
@@ -173,7 +173,7 @@ public CavernURLGenerator(String root) {
             }
 
             List<URL> baseURLs = getBaseURLs(target, protocol.getSecurityMethod(), scheme);
-            if (baseURLs == null) {
+            if (baseURLs == null || baseURLs.size() == 0) {
```

Comment:
my IDE always whines that I should use isEmpty() instead of size()==0 and it seems like a good idea

Reply:
Done